src/main/java/com/sourcegraph/config/CodyAuthNotificationActivity.java

```diff
@@ -79,15 +64,4 @@ public void actionPerformed(@NotNull AnActionEvent anActionEvent) {
     notification.addAction(neverShowAgainAction);
     Notifications.Bus.notify(notification);
   }
-
-  private void showMissingTokenNotification() {
```

Comment:
btw, I wonder... what if the token expires (or is removed)? 🤔 what will happen (with chat, with autocomplete)?

Reply:
I will test this scenario today. In general I think we should then (if user try to call some action explicitly) show cody panel, and it should be showing login panel.

Reply:
By the way, VSCode has various APIs for showing views, we don't implement them. If we need to show the view, we should see if the extension is trying to do it anyway. Maybe we just need to implement a couple of those methods/properties in the VSCode shim.